osu.Game/Screens/Play/Player.cs

```diff
@@ -385,14 +388,17 @@ private void updateBackgroundElements()
             var storyboardVisible = showStoryboard && beatmap.Storyboard.HasDrawable;
 
             storyboardContainer.FadeColour(new Color4(opacity, opacity, opacity, 1), 800);
-            storyboardContainer.FadeTo(storyboardVisible && opacity > 0 ? 1 : 0);
+            storyboardContainer.FadeTo(storyboardVisible && opacity > 0 ? 1 : 0, 800, Easing.OutQuint);
+            storyboardContainer.BlurTo(blur, 800, Easing.OutQuint);

```

Comment:
this is going to absolutely destroy performance on older devices, just so you're aware